docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -0,0 +1,99 @@
+= Customizing the build pipeline
+
+You can customize the way that {ProductName} builds components, to meet your specific needs, in two ways:
+
+* By changing the parameters of the build pipeline of a component
+* By extending the pipeline with your own tasks (or deleting existing tasks)
+
+For example, you can change for how long {ProductName} saves the container images of component, by changing the value of the `image-expires-after` parameter. Or you can ensure that the container images it builds for a component are compliant with regulations for your industry, by adding a compliance check as a task to its build pipeline. 
+
+.Prerequisites
+To customize a build pipeline, you must first xref:how-to-guides/configuring-builds/proc_upgrade_build_pipeline.adoc[upgrade it]. 
+
+.Procedure
+
+To customize the build pipeline in any way, start by completing the following steps:
+
+. In your preferred IDE, navigate to the `.tekton` directory in the repository of your component.
+
+. There should be two files, one ending with `push.yaml` and another ending with `pull-request.yaml`. Open both of them.
+ 
+
+== Changing parameters
+
+.Procedure
+
+. In each `.yaml` file in the `.tekton` directory, customize the parameters of the build pipeline by changing the pipeline `params`:
+.. Pipeline params have the paths `.spec.params` and `.pipelineSpec.params`
+.. You can change the default `value` of existing parameters. For example, you can set the value of `image-expires-after` to `2w`, instead of `5d` 
+.. You can also add parameters. New parameters must include a `name` and `value`.
+. Commit your changes to the repository of the component.
+
+
+== Extending the build pipeline with your own tasks
+
+.Procedure
+
+. In each `.yaml` file in the `.tekton` directory, add a new task under `tasks`. 
+
++ 
+Example task:
++
+[source]
+--
+name: example-task
+      params:
+      - name: example-param
+        value: “Example”
+      runAfter:
+      - build-container #You can be more specific by choosing another task
+      taskRef:
+        bundle: <path to container image> 
+        name: example-task
+      when:
+      - input: $(params.skip-checks)    #This references the pipeline parameters
+        operator: in
+        values:
+        - "false"
+--
++
+NOTE: If you want to define a task directly in this file, rather than using `taskRef`, you can use `taskSpec`. Visit the documentation linked at the end of this guide for further explanation.
```

Comment:
We tend to avoid `in this guide` So I'd say something like, 

Visit the documentation linked in the `Additional resources` section.